src/Hl7.Fhir.Support.Poco/Model/Date.cs

```diff
@@ -108,12 +109,11 @@ public string Value
         public DateTimeOffset? ToDateTimeOffset() =>
             Value == null ? null : (DateTimeOffset?)PrimitiveTypeConverter.ConvertTo<DateTimeOffset>(Value);
 
-        public Primitives.PartialDate? ToPartialDate() =>
-            Value != null ? (Primitives.PartialDate?)Primitives.PartialDate.Parse(Value) : null;
-
-        public Primitives.PartialDateTime? ToPartialDateTime() =>
-            Value != null ? (Primitives.PartialDateTime?)Primitives.PartialDateTime.Parse(Value) : null;
+        public P.Date ToPartialDate() =>
```

Comment:
Should this not be named ToDate() (we have no PartialDate anymore)

Reply:
done.